pandas/tests/indexes/multi/test_monotonic.py

```diff
@@ -176,7 +177,11 @@ def test_is_strictly_monotonic_decreasing():
     assert idx._is_strictly_monotonic_decreasing is False
 
 
-def test_is_monotonic_with_nans():
+@pytest.mark.parametrize(
+    "values",
+    [[(np.nan,), (1,), (2,)], [(1,), (np.nan,), (2,)], [(1,), (2,), (np.nan,)]],
+)
+def test_is_monotonic_with_nans(values):
     # GH: 37220
-    idx = pd.MultiIndex.from_tuples([(np.nan,), (1,)], names=["test"])
+    idx = pd.MultiIndex.from_tuples(values, names=["test"])
     assert idx.is_monotonic_increasing is False
```

Comment:
Could also parametrize over is_monotonic_increasing and is_monotonic_decreasing if indeed we think this should always return False

Reply:
Yes, makes sense. Done